README.md

```diff
@@ -479,10 +521,44 @@ process: true
 	- `process` - Whether to process the plate. Default is `true`. Set to `false` if this plate should not be processed.
 - `process` - Whether to process the batch. Default is `true`. Set to `false` if this batch should not be processed.
 
-# Rerunning the pipeline 
-- The instructions in this README assumes that the profiling pipeline is being run for the first time within the data repository. It is possible that you are rerunning the pipeline with a new config file in a repository that already contains profiles. In that case, after cloning the data repository, it is important to download the profiling-recipe submodule anddownload the profiles from Git LFS before running the pipeline. This can be done using the following commands
+# Rerunning the pipeline
+- The instructions in this README assumes that the profiling pipeline is being run for the first time within the data repository. It is possible that you are rerunning the pipeline with a new config file in a repository that already contains profiles. In that case, after cloning the data repository, it is important to download the profiling-recipe submodule and download the profiles from DVC before running the pipeline. This can be done using the following commands
 
 ```bash
 git submodule update --init --recursive
-git lfs pull
+dvc pull
 ```
+
+# Using DVC
+Additional information about using DVC that you may find useful:  
+When handling large files or a large folder, do NOT add them to GH with `git add`. Instead, add them to DVC with `dvc add`.
+This uploads the large file/folder to S3 and creates a pointer to that upload on S3 in the GH repo (that we track instead of the file/folder itself).
+It also updates .gitignore so that GH doesn't track the large file/folder itself.
+Then `dvc push` to upload the files to S3.
+
+```bash
+# Add a file or folder to DVC
+dvc add LARGEFILE.csv
+dvc push
+```
+Then add the .dvc version of the file/folder that is created to github along with the .gitignore. Commit.
+```bash
+git add LARGEFILE.csv.dvc
+git add .gitignore
+git commit -m "add largefile"
+```
+
+## Download data stored by DVC in S3:
+ ```bash
+# Download ALL data stored by DVC in S3
+# Only do this if you really, truly want ALL the data
+ dvc pull
+ ```
+ ```bash
+ # Download a specific file stored by DVC in S3
+ dvc get https://github.com/ORGANIZATION/DATA-REPO.git LARGEFILE.csv
```

Comment:
@ErinWeisbart consider clarifying that it should be the relative path to `LARGEFILE.csv` (and not just `LARGEFILE.csv`)?

Reply:
Thanks @shntnu. Fixed.